src/Template.php

```diff
@@ -25,13 +25,13 @@ public function __construct(Kirby $kirby, string $name, string $type = 'html', s
 
     public function render(array $data = []): string
     {
-        if ($this->isBlade() && $this->hasDefaultType() === true) {
+        if ($this->isBlade()) {
             View::share('kirby', $data['kirby']);
             View::share('site', $data['site']);
             View::share('pages', $data['pages']);
             View::share('page', $data['page']);
 
-            $html = View::make($this->name, $data)->render();
+            $html = View::file($this->bladeFile(), $data)->render();
```

Comment:
Can we use the existing method? I tried it out in a little example and it works for me.

```suggestion
            $html = View::file($this->file(), $data)->render();
```